autorest/adal/persist.go

```diff
@@ -25,6 +25,24 @@ func LoadToken(path string) (*Token, error) {
 	return &token, nil
 }
 
+// LoadCLITokens restores a set of AzureCLIToken objects from a file located at 'path'.
+func LoadCLITokens(path string) (*[]AzureCLIToken, error) {
```

Comment:
Only because I wanted to return a `nil` object when an error is returned (to match the `LoadToken` function above - I can switch this out, but returning an empty object feels like an unexpected response IMO?